pages/common/duc.md

```diff
@@ -0,0 +1,28 @@
+# duc
+
+> Duc: index, query and graph disk usage
```

Comment:
Could you maybe think of a better description than this, because its a little vague.

Reply:
Done